### PR TITLE
4 and 5 added

### PR DESCRIPTION
**Pull request by rohitrango**
#1  Added 4 and 5 for now ... Tested using interactive mode, everything seems fine.

By mistake I had also removed spaces after z and 0 which had some issues, but restored it back :P

### CODE REVIEW

File: 3d_font.txt [hunk 1]
```diff
@@ -363,20 +363,20 @@ z
...
-    
-     
-      
-       
-        
-         
-          
-           
-            
+
```
Comment (udiboy1209) [inline 3d_font.txt]: These lines had trailing spaces which you removed :P. Could you remove this change from the commit?

**Comment by rohitrango** [comment]
On it !

File: 3d_font.txt [hunk 2]
```diff
@@ -433,3 +433,31 @@ z
         | \________/  /
         \ |        | /
          \|________|/
+4
+ ____       ____
```
Comment (udiboy1209) [inline 3d_font.txt]: This width is 16 characters, make it 14 characters.

File: 3d_font.txt [hunk 3]
```diff
@@ -433,3 +433,31 @@ z
...
+  \  \___________   \
+   \ |          |\   \
+    \|__________| \   \
+                \  \   \
+                 \  \   \
+                  \  \___\
+                   \ |   |
+                    \|___|
+5
+   ___________________
```
Comment (udiboy1209) [inline 3d_font.txt]: same here. 20 characters! looks too big. Maintain 14 chars, I don't see why this will give a problem.

File: 3d_font.txt [hunk 4]
```diff
@@ -433,3 +433,31 @@ z
...
+  /                   \
+ |\    ________________\
+ | \   \               |
+ \  \   \______________|__
+  \  \                    \
+   \  \_________________   \
+    \ |                /   /|
+     \|__ ____________/   / |
```
Comment (udiboy1209) [inline 3d_font.txt]: This slant at the edge is 4 lines long, which isn't matching with other characters. Try reducing it to 3 lines

**Comment by udiboy1209** [comment]
Merged by [3aeb3a](https://github.com/udiboy1209/3dwrite/commit/3aeb3a5e8e7dd75192982e7128f8946240d140af)

Really nice contribution! Thanks :tada:

**Comment by rohitrango** [comment]
My first PR merged :p

**Pull request closed without merging**